src/database/exiles_database.py

```diff
@@ -107,27 +134,29 @@ def get_user_exiles(user_id) -> List[tuple]:
         cursor.execute(query, params)
         res = cursor.fetchall()
 
-        return res
+        if res is not None:
+            return [Exile(*row) for row in res]  # Create a list of Exile objects
+        else:
+            return []
 
 
-def get_user_active_exile(user_id) -> PendingExile:
+def get_all_active_exiles() -> list[Exile]:
     conn = DatabaseConnection()
 
     with conn.get_cursor() as cursor:
         query = """
-        SELECT e.exileID, u.userID, u.discordUserID, e.endTimestamp
+        SELECT e.exileID, u.userID, u.discordUserID, e.reason, e.exileStatus, e.startTimestamp, e.endTimestamp
         FROM exiles e
         JOIN users u ON e.userID = u.userID
-        WHERE u.userID = %s AND  e.exileStatus = %s 
-        LIMIT 1;
+        WHERE e.exileStatus = %s AND e.reason != 'roulette';
         """
 
-        params = (user_id, ExileStatus.TIMED_EXILED)
+        params = (ExileStatus.TIMED_EXILED,)
```

Comment:
i dont think we need the parentheses and the comma

Reply:
We do, params needs to be a tuple even if it's one item, and the parens plus comma is how to make a tuple with a single value

Reply:
ew that is disgusting lol

Reply:
what's worse is when you accidentally make a non-tuple into a tuple because of random parans or commas, then wonder why your code is fucked. Nova did that exact thing in this PR initially lol